source/deimos/openssl/evp.di

```diff
@@ -912,8 +912,27 @@ alias EVP_seed_cfb = EVP_seed_cfb128;
 const(EVP_CIPHER)* EVP_seed_ofb();
 }
 
-void OPENSSL_add_all_algorithms_noconf();
-void OPENSSL_add_all_algorithms_conf();
+static if (OPENSSL_VERSION_BEFORE(1, 1, 0, 0))
+{
+    void OPENSSL_add_all_algorithms_noconf();
+    void OPENSSL_add_all_algorithms_conf();
+}
+else
+{
+    auto OPENSSL_add_all_algorithms_conf()()
+    {
+		pragma(inline, true);
```

Comment:
Why the pragma inline ? Also indent is off.

Reply:
Fixed indent (lots of tabs all around the place for whatever reason, I guess the original bindings author's editor inserted them).

Inline is explained in commit message.